music21/roman.py

```diff
@@ -3089,6 +3091,28 @@ def _updatePitches(self):
                 f'_updatePitches() was unable to derive pitches from the figure: {self.figure!r}'
             )  # pragma: no cover
 
+    def transpose(self, value, *, inPlace=False) -> t.Optional[T]:
```

Comment:
for this to work, self has to be annotated with T as well.

Reply:
see Harmony L 2338

Reply:
OK, fixed, but mypy didn't seem to be complaining. The "x" seems to be because coverage decreased slightly. (Not quite sure why since both `inPlace=True` and `inplace=False` are tested in the `RomanNumeral.transpose()` doctest.)